### ISSUE TITLE
"Xamarin.Forms" checkbox of the Create New Workbook window is not accessible by Keyboard.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#254]
### Steps to Reproduce

1. Launch Xamarin Workbook application.
2. Change the Platfrom to IOS/Android from the create a new workbook window.
3. Navigate to "Xamarin.Forms" checkbox using keyboard.
4. Observe the keyboard focus on this "Xamarin.Forms" checkbox.

### Expected Behavior
User should be able to navigate to "Xamarin.Forms" checkbox.

### Actual Behavior
"Xamarin.Forms" is not accessible by keyboard.

### User Impact:
Keyboard user won't be able to access this "Xamarin.Forms" checkbox, so user might be blocked here.

### Log File

![kb4_xamarincheckbox](https://user-images.githubusercontent.com/22956130/39687277-2736a5ea-51bd-11e8-83d8-02addefd6be2.png)

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1814
